src/transformations/visual.jl

```diff
@@ -2,12 +2,14 @@ struct Visual
     plottype::PlotType
     attributes::NamedArguments
 end
-Visual(plottype=Plot{Any}; kwargs...) = Visual(plottype, NamedArguments(kwargs))
+Visual(plottype::PlotType=Plot{Any}; kwargs...) = Visual(plottype, NamedArguments(kwargs))
```

Comment:
Ah here it is!!! I was searching where this got initialized... 
I think this needs to be Plot{plot} now!